app/models/distribution.rb

```diff
@@ -71,6 +71,8 @@ class Distribution < ApplicationRecord
           start_date: Time.zone.today.beginning_of_week.beginning_of_day, end_date: Time.zone.today.end_of_week.end_of_day)
   end
 
+  scope :for_year, ->(year) { where("extract(year from issued_at) = ?", year) }
```

Comment:
I bet this (and some similar methods) would be handy to put into a Concern. The app does a lot with date-based-retrieval of resources